app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md

```diff
@@ -15,4 +15,4 @@ You’ll need a degree and [qualified teacher status](/what-is-qts) (QTS) to wor
 
 [Teacher training courses for graduates](https://www.find-postgraduate-teacher-training.service.gov.uk/) may also work for you. If you have knowledge and experience in a particular subject area there are specific training courses you can choose.
 
-If you're concerned about your subject knowledge, you may also be able to do a ['subject knowledge enhancement' (SKE) course](/what-subject-do-you-want-to-teach). This can help if you have an unrelated degree but relevant professional experience, or you haven't used your subject knowledge for a while.
+If you're concerned about your subject knowledge, you may also be able to do a <a href="/train-to-be-a-teacher/what-subject-do-you-want-to-teach">'subject knowledge enhancement' (SKE) course</a>. This can help if you have an unrelated degree but relevant professional experience, or you haven't used your subject knowledge for a while.
```

Comment:
@EllieNodder - totally inconsequential, but did the markdown format link not work here? The quote symbols getting in the way?

Reply:
yeah it wasn't pulling through properly